### PR TITLE
L10n rc other compensation

### PR DESCRIPTION
**Pull request by BarryPiccinni**
PR Includes various improvements which have been tested and signed off.

Improvements include:

- Contextualise offender screens
- Contextualise other compensation screens
- Contextualise victim details
- GP and Dentists adresss screens changed from "your gp/dentist" to "the gp/dentist"
- Describe the crime no longer requires a contextualised title.
- Age restrictions pertaining to the victim now only apply to physical injury journeys

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9875,35 +10120,6 @@ module.exports = {
...
-                    {
-                        language: 'en',
-                        namespace: 'p-applicant-describe-incident',
-                        resources: {
-                            'q-applicant-describe-incident': {
-                                error: {
-                                    required:
-                                        'Select yes if you would like to describe the crime in your own words',
-                                    'required_someone-else':
-                                        'Select yes if you would like to briefly describe the crime'
```
Comment (armoj) [inline index.js:9897]: have they specifically asked that this schema is reverted?

File: index.js [hunk 2]
```diff
@@ -9939,42 +10155,16 @@ module.exports = {
...
-                ]
-            },
             schema: {
                 $schema: 'http://json-schema.org/draft-07/schema#',
                 type: 'object',
                 required: ['q-applicant-incident-description'],
                 properties: {
                     'q-applicant-incident-description': {
                         type: 'string',
-                        title: 'l10nt:q-applicant-incident-description.title{?lng,context,ns}',
```
Comment (armoj) [inline index.js:9975]: same as above. These were different in the prototype (for minors, compared to self journey). Has Julie etc. just asked that we leave these ones?

**Review by armoj — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (BarryPiccinni) [reply index.js:9897]: Yeah, the question should always be asked to the applicant, so not necessary to contextualise it.

File: index.js (hunk 2 above)
Comment (BarryPiccinni) [reply index.js:9975]: That was an oversight, Julie said they should be consistent across both journeys, not different.

**Review by armoj — APPROVED** [review]

**Pull request merged**